charts/modernization-api/templates/deployment.yaml

```diff
@@ -62,9 +58,13 @@ spec:
             '--nbs.ui.features.pageBuilder.page.management.edit.enabled={{ .Values.pageBuilder.page.management.edit.enabled }}',
             '--nbs.ui.features.search.view.enabled={{ ((((.Values).ui).search).view).enabled | default "false" }}',
             '--nbs.ui.features.search.view.table.enabled={{ (((((.Values).ui).search).view).table).enabled | default "false" }}',
-            '--nbs.ui.features.patient.add.extended.enabled={{ (((((.Values).ui).patient).add).extended).enabled | default "false" }}',
-            '--nbs.ui.settings.smarty.key={{ .Values.ui.smarty.key }}',
-            '--nbs.ui.settings.analytics.key={{ .Values.ui.analytics.key }}'
+            '--nbs.ui.features.search.events.enabled={{ ((((.Values).ui).search).events).enabled }}',
```

Comment:
There should be a default value for new and existing inputs that are required in values.yaml (the de facto release values file). Otherwise deployments may break in other environments.

Reply:
Made sure that boolean values have a default so that the service starts as expected.  Will look into making blank values for boolean valid in the future